core/src/main/scala/zio/jdbc/ZConnectionPool.scala

```diff
@@ -45,7 +45,6 @@ object ZConnectionPool {
     ZLayer {
       for {
         _      <- ZManaged.attempt(Class.forName("org.h2.Driver"))
-        config <- ZManaged.service[ZConnectionPoolConfig]
```

Comment:
getting the config for the distinct pools seems necessary as the config is once again pulled out of the environment in the `make` call.